examples/ERC20/index.rsh

```diff
@@ -0,0 +1,146 @@
+// ERC-20 Specification
+// https://eips.ethereum.org/EIPS/eip-20
+//
+// ERC-165
+// https://eips.ethereum.org/EIPS/eip-165
+'reach 0.1';
+
+const ERC165 = mixin({
+  IDs: [ Bytes.fromHex('0x01ffc9a7'), ],
+  View: [{
+    supportsInterface: Fun([Bytes(4)], Bool),
+  }],
+});
+
+const ERC20 = mixin({
+  //IDs: [ Bytes.fromHex('0xTODO'), ],
+  View: [{
+    name: Fun([], StringDyn),
+    symbol: Fun([], StringDyn),
+    // TODO Decimals should return uint8
```

Comment:
💩

Reply:
Ah, yeah, I wasn't sure about that.  After I checked the ABI encoding to check that using UInt for uint8 would be fine, I forgot to edit the comment.